src/create.cpp

```diff
@@ -463,6 +463,15 @@ namespace create {
     sideMotorPower = roundf(side * 127);
     vacuumMotorPower = roundf(vacuum * 127);
 
+    if (model.getVersion() == V_1) {
+        uint8_t cmd[2] = { OC_MOTORS,
+                           (uint8_t)((side != 0.0 ? 1 : 0) |
```

Comment:
nitpick: Use a static_cast for consistency with the other parts of the code.
```suggestion
                           static_cast<uint8_t>((side != 0.0 ? 1 : 0) |
```